### PR TITLE
+ Actions on groups

### PR DESCRIPTION
**Pull request by smarinier**
- Add "+" actions on Workspace Groups and Added Groups
- + Vue Warning : use name instead of title

### CODE REVIEW

File: src/SpaceMenuItem.vue [hunk 1]
```diff
@@ -24,22 +24,55 @@
...
+				</NcActionText>
+				<NcActionInput v-show="true"
+					ref="createGroupInput"
+					:class="'ws-modal-action'"
+					icon="icon-group"
+					:close-after-click="true"
+					:show-trailing-button="true"
+					@submit="onNewWorkspaceGroup">
+					{{ t('workspace', 'Group name') }}
+				</NcActionInput>
```
[the remark this answers is not in the document]
Comment (smarinier) [reply src/SpaceMenuItem.vue:52]: ?? I made it like SpaceDetails. What is the gain or difference ?

[the remark this answers is not in the document]
Comment (zak39) [reply src/SpaceMenuItem.vue:52]: Not exactly, you defined a Text associated with an input field.

In SpaceDetails, we have one field and when we clicked on it, another input field appears.

[Capture vidéo du 2024-08-23 11-25-34.webm](https://github.com/user-attachments/assets/f73c20a6-a8f8-4b23-ac58-527f1459a0b7)

[the remark this answers is not in the document]
Comment (smarinier) [reply src/SpaceMenuItem.vue:52]: Indeed. This was made on purpose after a discussion with Dorianne (or Clémentine).
After discussion:
- the autoreplace take sense in the menu, but not really here, as it is a one click action (and not a menu), even if it uses the drop menu.
- this let the user know what he is doing (create a workspace group)
- this makes a gain a one click, and that is in general a better option in U/X

[the remark this answers is not in the document]
Comment (smarinier) [reply src/SpaceMenuItem.vue:52]: We can see (i'm just see this for another subject), that NextCloud keep the action name in "Users"
image

**Pull request merged**